content/docs/user-guide/experiment-management/index.md

```diff
@@ -1,113 +1,82 @@
-  cleans the slate so you can repeat the process.
-
-## Checkpoints in source code
+_New in DVC 2.0 (see `dvc version`)_
 
-To track successive steps in a longer experiment, you can register checkpoints
-from your code at runtime. This allows you, for example, to track the progress
-in deep learning techniques such as evolving neural networks.
+DVC experiment management features build on top of base DVC features to form a
+comprehensive framework to organize, execute, manage, and share ML experiments.
+They support support these main approaches:
 
-This kind of experiments track a series of variations (the checkpoints) and its
-execution can be stopped and resumed as needed. You interact with them using
-`dvc exp run` and its `--rev`, `--reset` options (see also the `checkpoint`
-field in `dvc.yaml` `outs`).
+- Compare params and metrics of existing project versions (for example different
+  Git branches) against each other or against new results in your workspace
+  (without committing them).
 
-> 📖 To learn more, see the dedicated
-> [Checkpoints](/doc/user-guide/experiment-management/checkpoints) guide.
+- [Run and capture] multiple experiments (derived from any project version as
+  baseline) without polluting your Git history. DVC tracks them for you, letting
+  you compare and share them. 📖 More info in the [Experiments
+  Overview][experiments].
 
-## Persistent experiments
+- Generate [checkpoints] during your code execution to analyze the internal
+  progress of deep experiments. DVC captures [live metrics](/doc/dvclive) at
+  runtime, and lets you manage them in batches.
 
-When your experiments are good enough to save or share, you may want to store
-them persistently as Git commits in your <abbr>repository</abbr>.
+[run and capture]: /doc/user-guide/experiment-management/running-experiments
+[experiments]: /doc/user-guide/experiment-management/experiments-overview
+[checkpoints]: /doc/user-guide/experiment-management/checkpoints
 
-Whether the results were produced with `dvc repro` directly, or after a
-`dvc exp` workflow (refer to previous sections), the `dvc.yaml` and `dvc.lock`
-pair in the <abbr>workspace</abbr> will codify the experiment as a new project
-version. The right <abbr>outputs</abbr> (including
-[metrics](/doc/command-reference/metrics)) should also be present, or available
-via `dvc checkout`.
+> 👨‍💻 See [Get Started: Experiments](/doc/start/experiments) for a hands-on
+> introduction to DVC experiments.
 
 ### Organization patterns
 
-DVC takes care of arranging `dvc exp` experiments and the data
-<abbr>cache</abbr> under the hood. But when it comes to full-blown persistent
-experiments, it's up to you to decide how to organize them in your project.
-These are the main alternatives:
+It's up to you to decide how to organize completed experiments. These are the
+main alternatives:
 
 - **Git tags and branches** - use the repo's "time dimension" to distribute your
```

Comment:
I think `dvc exp` commands work towards this organization pattern, not separate directories. IMO we can modify this section to describe the organization pattern DVC leads. We don't have much facility to use "space dimension" for experiments.

Reply:
This index tries to cover the traditional experiments as well. It's not exclusively about `dvc exp` until you get to the sub-pages. That's one of the reasons for creating a separate DVC Experiments overview page in this PR.

Reply:
It would be great to improve on this indeed so it's clear that we're separating manual exp tracking you can do on DVC projects vs. the DVC Experiments workflow. Similar to what's described in https://www.notion.so/iterative/Experiment-Workflows-3873c4f3cc2e49c6a7871a831bc8302b ⌛

Reply:
> When I look into the text, I think DVC helps to organize the experiments in "space dimension" as well.

It can, for example multiple via `dvc.yaml` files (copy/pasted + small changes) or via `init --subrepo` (monorepo structure). Both supported by DVC

Reply:
p.s. this makes me thing about another route: using `foreach` stages to quickly define multiple experiments based on a params file and running them all (in parallel). I guess it's a pre-`exp` way to manage experiments with DVC, but in which you can see all the results at once in your workspace (may be messy unless you create a bunch of subdirectories so perhaps it's the same as the  "space dimension")...

Reply:
p.p.s I added a section about custom labels as well (for org pattern) based on [this table](https://docs.google.com/presentation/d/1C_owNoC72GvrpyMGlonHEYJ9I2rl2SLHkZQDMx0eT7A/edit#slide=id.gcb78e52e40_0_635). See c68fc784

Reply:
> It can @iesahin, for example multiple via dvc.yaml files (copy/pasted + small changes) or via init --subrepo (monorepo structure). Both supported by DVC

The `dvc exp` workflow is orthogonal to these features, they are not alternatives to each other. One can use multiple `dvc.yaml` files with `dvc exp` as well.

Reply:
They are conceptual alternatives which you can combine, which is already stated in the text.